content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -66,47 +68,93 @@ type Props = {
   stops: [Stop];
 };
 
+const Stop: FC<{ stop: Stop }> = ({ stop }) => {
+  const { isEnhanced } = useContext(AppContext);
+  const hasShowFullTranscriptionButton =
+    (stop.transcription?.length || 0 > 1) && isEnhanced;
+  const transcriptionFirstParagraph = stop.transcription?.slice(0, 1);
```

Comment:
I think it could be clearer that we show a full transcription if there are multiple paragraphs, either a comment or:

```suggestion
  const transcriptionFirstParagraph = stop.transcription?.slice(0, 1);
  const hasShowFullTranscriptionButton =
    (stop.transcription != transcriptionFirstParagraph) && isEnhanced;

```

Reply:
It looks like the most appealing option, but I don't think `stop.transcription != transcriptionFirstParagraph` will work like we want – you have to do a bit of a dance to compare arrays in JS – comment maybe makes more sense.